app/views/casework/crime_applications/sections/_premium_bonds_partner.html.erb

```diff
@@ -1,20 +1,35 @@
-<% unless details.partner_has_premium_bonds.nil? %>
-  <%= govuk_summary_card(title: label_text(:premium_bonds_partner)) do %>
+<% unless details.has_premium_bonds.nil? && details.partner_has_premium_bonds.nil? %>
+  <%= govuk_summary_card(title: label_text(:premium_bonds)) do %>
     <%= govuk_summary_list(actions: false) do |list|
           list.with_row do |row|
-            row.with_key { label_text(:has_premium_bonds) }
+            row.with_key { label_text(:has_premium_bonds_client) }
+            row.with_value { value_text(details.has_premium_bonds) }
+          end
+
+          if details.has_premium_bonds == 'yes'
+            list.with_row do |row|
+              row.with_key { label_text(:premium_bonds_holder_number_client) }
+              row.with_value { details.premium_bonds_holder_number }
+            end
+          end
+
+          list.with_row do |row|
+            row.with_key { label_text(:has_premium_bonds_partner) }
             row.with_value { value_text(details.partner_has_premium_bonds) }
           end
 
           if details.partner_has_premium_bonds == 'yes'
             list.with_row do |row|
-              row.with_key { label_text(:premium_bonds_holder_number) }
+              row.with_key { label_text(:premium_bonds_holder_number_partner) }
               row.with_value { details.partner_premium_bonds_holder_number }
             end
+          end
 
+          if details.has_premium_bonds == 'yes' || details.partner_has_premium_bonds == 'yes'
             list.with_row do |row|
+              value = details.premium_bonds_total_value.to_i + details.partner_premium_bonds_total_value.to_i
```

Comment:
We also need to add this up to pass to MAAT (arguably a problem for their adaptor, but realistically probably just depends who has capacity), if we store the combined amount in the datastore, we can display that here directly